x-pack/test/detection_engine_api_integration/basic/tests/query_signals.ts

```diff
@@ -22,23 +22,6 @@ export default ({ getService }: FtrProviderContext) => {
 
   describe('query_signals_route and find_alerts_route', () => {
     describe('validation checks', () => {
-      it('should not give errors when querying and the signals index does not exist yet', async () => {
```

Comment:
deleted this test because the index is installed on Kibana start instead of on first rule execution so it should always exist when the signals are queried.